.rubocop.yml

```diff
@@ -47,6 +47,7 @@ Style/BracesAroundHashParameters:
 Style/ClassAndModuleChildren:
   Description: Compact style reduces the required amount of indentation.
   EnforcedStyle: compact
+  Enabled: false
```

Comment:
This change is required because of the module declaration changes that I will comment on in that file.